src/lesson_6/script.js

```diff
@@ -0,0 +1,104 @@
+function minimum (a, b, c, d) {
+  if(! Number.isInteger(a) || !Number.isInteger(b) || !Number.isInteger(c) || !Number.isInteger(d)) {
+      console.log('input parameters are invalid');
```

Comment:
To make it shorter here and below
```suggestion
      return console.log('input parameters are invalid');
```